OneSignalExample/Assets/OneSignal/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
```

Comment:
This section is intend by a space for some reason, where the other sections are not.

Reply:
Seems this is no longer an issue after the rebase